dspace-server-webapp/src/test/java/org/dspace/app/rest/AuthorizationRestRepositoryIT.java

```diff
@@ -1517,6 +1525,873 @@ public void findByObjectAndFeatureInternalServerErrorTest() throws Exception {
+            .andExpect(jsonPath("$._embedded.authorizations", containsInAnyOrder(
+                allOf(
+                    hasJsonPath("$.id", is(
+                        eperson.getID().toString() + "_"
+                            + alwaysTrue.getName() + "_"
+                            + comRest.getUniqueType() + "_" + comRest.getId()
+                    )),
+                    hasJsonPath("$.type", is("authorization")),
+                    hasJsonPath("$._embedded.feature.id", is(alwaysTrue.getName())),
+                    hasJsonPath("$._embedded.eperson.id", is(eperson.getID().toString()))
+                ),
+                allOf(
+                    hasJsonPath("$.id", is(
+                        eperson.getID().toString() + "_"
+                            + trueForLoggedUsers.getName() + "_"
+                            + comRest.getUniqueType() + "_" + comRest.getId()
+                    )),
+                    hasJsonPath("$.type", is("authorization")),
+                    hasJsonPath("$._embedded.feature.id", is(trueForLoggedUsers.getName())),
+                    hasJsonPath("$._embedded.eperson.id", is(eperson.getID().toString()))
+                ),
+                allOf(
+                    hasJsonPath("$.id", is(
+                        eperson.getID().toString() + "_"
+                            + alwaysTrue.getName() + "_"
+                            + secondComRest.getUniqueType() + "_" + secondComRest.getId()
+                    )),
+                    hasJsonPath("$.type", is("authorization")),
+                    hasJsonPath("$._embedded.feature.id", is(alwaysTrue.getName())),
+                    hasJsonPath("$._embedded.eperson.id", is(eperson.getID().toString()))
+                ),
+                allOf(
+                    hasJsonPath("$.id", is(
+                        eperson.getID().toString() + "_"
+                            + trueForLoggedUsers.getName() + "_"
+                            + secondComRest.getUniqueType() + "_" + secondComRest.getId()
+                    )),
+                    hasJsonPath("$.type", is("authorization")),
+                    hasJsonPath("$._embedded.feature.id", is(trueForLoggedUsers.getName())),
+                    hasJsonPath("$._embedded.eperson.id", is(eperson.getID().toString()))
+                )
+            )));
+
+        // verify that it works for administators inspecting other users - by assuming login
+        getClient(adminToken).perform(get("/api/authz/authorizations/search/objects")
+            .param("type", "core.community")
+            .param("uuid", comId)
+            .param("uuid", secondComId)
+            .param("projection", "level")
+            .param("embedLevelDepth", "1")
+            .param("feature", alwaysTrue.getName())
+            .param("feature", alwaysFalse.getName())
+            .param("feature", trueForAdmins.getName())
+            .header("X-On-Behalf-Of", eperson.getID()))
```

Comment:
Just to ensure this call is equivalent to the previous call, we may want to also add the `trueForLoggedInUsers` feature to the list.  That way we prove this works the exact same as the previous one.